build/configure

```diff
@@ -5,6 +5,7 @@ base_dir=`(cd ..; pwd -P)`
 scxomi_dir=`(cd ../../omi/Unix; pwd -P)`
 scxpal_dir=`(cd ../../pal; pwd -P)`
 scxcim_dir=`(cd ../../opsmgr; pwd -P)`
+auoms_dir=`(cd ../../auoms; pwd -P)`
```

Comment:
you mentioned that there are two packages that need to sit under /data folder . I do not see them being installed through configure or it is done somewhere else?